### PR TITLE
[All hosts] (auth) remove link to sample that is going away

### PR DESCRIPTION
**Pull request by Rick-Kirkham**

### CODE REVIEW

**Comment by o365devx** [comment]
OPS Build status updates of commit _[19bb42b](https://github.com/OfficeDev/office-js-docs-pr/pull/750/commits/19bb42bc04ffbd6cbdf82c12bcedca9337ff5d40)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[19bb42b](https://github.com/OfficeDev/office-js-docs-pr/pull/750/commits/19bb42bc04ffbd6cbdf82c12bcedca9337ff5d40)_: 

:clock10: **Incremental building**: average incremental building time is _29 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[19bb42b](https://github.com/OfficeDev/office-js-docs-pr/pull/750/commits/19bb42bc04ffbd6cbdf82c12bcedca9337ff5d40)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/dialog-api-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/auth-sample-references/docs/develop/dialog-api-in-office-add-ins.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201902132303351931-750%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=y1qtT85amnMwxXvrZbvjZHeIyaBe6auZT1i2irHIqDU%3D&st=2019-02-13T22%3A59%3A52Z&se=2019-03-16T23%3A04%3A52Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/develop/dialog-api-in-office-add-ins.md [hunk 1]
```diff
@@ -394,10 +394,9 @@ The following is a simple and typical authentication flow:
 3. When the redirectPage.html page opens, it calls `messageParent` to report the success or failure to the host page and optionally also report user data or error data.
 4. The `DialogMessageReceived` event fires in the host page and its handler closes the dialog window and optionally does other processing of the message.
 
-For sample add-ins that use this pattern, see:
+For a sample add-in that use this pattern, see:
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/dialog-api-in-office-add-ins.md]:
```suggestion
For a sample add-in that uses this pattern, see:
```

**Comment by o365devx** [comment]
OPS Build status updates of commit _[31aca86](https://github.com/OfficeDev/office-js-docs-pr/pull/750/commits/31aca8605dcddb188d89420e5dcebd2f881a27eb)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[31aca86](https://github.com/OfficeDev/office-js-docs-pr/pull/750/commits/31aca8605dcddb188d89420e5dcebd2f881a27eb)_: 

:clock10: **Incremental building**: average incremental building time is _29 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[31aca86](https://github.com/OfficeDev/office-js-docs-pr/pull/750/commits/31aca8605dcddb188d89420e5dcebd2f881a27eb)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/dialog-api-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/auth-sample-references/docs/develop/dialog-api-in-office-add-ins.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201902132321375637-750%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=g6nJgYi3ScIlphRZSOPHCblRmZD7OYbGTr2wyXnDZTc%3D&st=2019-02-13T23%3A17%3A55Z&se=2019-03-16T23%3A22%3A55Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**